Upstream-Status: Inappropriate [ Cross compile specificated ]

OCF_ROOT_DIR has a default value and we use it. There has a line of code to
re-evaluate OCF_ROOT_DIR. But it is not applicable for cross compile and then
get empty value. So remove this line of code.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
diff --git a/configure.ac b/configure.ac
index 5b4fe5b..683504f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -406,7 +406,7 @@ dnl Variables needed for substitution
 AC_CHECK_HEADERS(heartbeat/glue_config.h)
 
 if test "$ac_cv_header_heartbeat_glue_config_h" = "yes"; then
-  OCF_ROOT_DIR=`extract_header_define heartbeat/glue_config.h OCF_ROOT_DIR`
+  :
 else
   enable_libnet=no
 fi
